src/niswitch/system_tests/test_system_niswitch.py

```diff
@@ -194,6 +195,7 @@ def _get_grpc_server_exe(self):
     def grpc_channel(self):
         server_exe = self._get_grpc_server_exe()
         proc = subprocess.Popen([str(server_exe)])
+        time.sleep(3)
```

Comment:
Since this is specifically called out as to be removed by the TODO comment and the other merged changes don't have a specific comment I'm leaving as is